content/docs/architecture/branch-protection.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Branch protection"
+date: 2020-10-05T11:14:39-04:00
+draft: false
+---
+## What is branch protection?
+
+Branch protection is a repository setting that enforces certain rules when trying to merge to a protected branch. See
+the [GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/about-protected-branches)
+for more details.
+
+## What do we use it for?
+
+We use it to enforce that mandatory jobs succeed before allowing a merge. These jobs are marked as `Required` in their GitHub
+status context. If you open a new Pull Request to a branch that has required job, these jobs will initially be marked as
+`Required - Context Expected`. This status is generated by GitHub and means that Prow didn't report the job as running yet.
+
+Additionally, the branch protection settings are interpreted by the mergeautomation
+[tide](https://github.com/kubernetes/test-infra/blob/master/prow/cmd/tide/README.md) in order to determine if jobs that
+are not managed by Prow should be considered mergeblocking.
+
+## How do we set up branch protection?
+
+Branch protection is configured by a job that runs periodically every six hours. You can see
```

Comment:
Right now you answer the question in the last part of this paragraph. Can we flip that and be more clear?

```
## How do we set up Branch Protection?

Branch Protection requires no manual set-up for repositories with pre-submit jobs. When a mandatory pre-submit job is added or removed from a repository, automation will ensure that the GitHub Branch Protection settings are updated as well. Branch Protection automation runs periodically, every six hours, so it may take some time for changes to jobs to be reflected. See the history of the job [here](link).
```